source/tutorials/metadata.rst

```diff
@@ -0,0 +1,138 @@
+Metadata in QIIME 2
+===================
+
+Metadata enables users of QIIME 2 to supplement their analyses with information relevant to their study. This manifests itself in many ways: indicating the barcode sequence when demultiplexing sequences, sorting and grouping taxonomic assignments in the taxonomic bar plots, or exploring diversity metrics with visualizations like ``alpha-group-significance`` or ``beta-group-significance``. In QIIME 1, metadata (also known as the metadata mapping file), was a user-defined file that contained these study-specific fields. QIIME 2 expands upon this idea, allowing users to provide their own study metadata (via a TSV file), or, by viewing other study-specific artifacts as metadata. Examples of both are presented in the following.
```

Comment:
I like that --- I will update the text to include these changes! To clarify on the first point though, "supplement" when used as a verb can mean "to complete," (how I was interpreting this) in addition to "adding to" (how I think you are interpreting it). Just wanted to point out that we are on the same page, but if this wasn't clear to you initially, I imagine it won't be clear to users, either! Thanks! 🎉